serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -273,25 +273,45 @@ When using the Kubernetes service discovery feature, you need to balance if your
 
 If the URI pattern is not found in the application properties, then discovery is not triggered. However, the scanning is performed anyway. Therefore, a short time is required to go through the startup scan.
 
+=== Available service discovery implementations
+
+Currently, there are two implementations available.
+
 You can enable the Kubernetes service discovery by adding a service discovery implementation to the application's dependencies as shown in the following Maven dependencies:
 
+==== `org.kie.kogito:kogito-addons-quarkus-fabric8-kubernetes-service-catalog`
+
+When enabled, it uses the Kubernetes Java API to discover the services.
+
 .Example Maven dependencies
 [source,xml]
 ----
 <dependency>
-    <!-- Service Discovery -->
-    <dependency>
-      <groupId>org.kie.kogito</groupId>
-      <artifactId>kogito-addons-quarkus-kubernetes</artifactId>
-    </dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-kubernetes</artifactId>
+</dependency>
+<dependency>
     <groupId>org.kie.kogito</groupId>
     <artifactId>kogito-addons-quarkus-fabric8-kubernetes-service-catalog</artifactId>
 </dependency>
 ----
 
-=== Available service discovery implementations
+==== `org.kie.kogito:kogito-addons-quarkus-microprofile-config-service-catalog`
```

Comment:
Same here about the description and expansion on the usage scenario.

Reply:
Done.